pkg/eventprocessor/eventfeed/impl/eventfeed.go

```diff
@@ -165,7 +165,8 @@ func (ef *EventFeed) Start(
 				ef.log.Warn().Err(err).Msgf("filter logs from %d to %d", fromHeight, toHeight)
 				if strings.Contains(err.Error(), "read limit exceeded") ||
 					strings.Contains(err.Error(), "Log response size exceeded") ||
-					strings.Contains(err.Error(), "is greater than the limit") {
+					strings.Contains(err.Error(), "is greater than the limit") ||
+					strings.Contains(err.Error(), "eth_getLogs and eth_newFilter are limited to a 10,000 blocks range") {
 					ef.maxBlocksFetchSize = ef.maxBlocksFetchSize * 80 / 100
```

Comment:
Quicknode only allows block range no bigger than 10k. If the event feed hits the limit, this is the part where the block range will be reduced until the request succeeds.